src/jit/codegenarm64.cpp

```diff
@@ -2896,6 +2911,11 @@ void CodeGen::genCodeForCpObj(GenTreeObj* cpObjNode)
     // While we normally update GC info prior to the last instruction that uses them,
     // these actually live into the helper call.
     gcInfo.gcMarkRegSetNpt(RBM_WRITE_BARRIER_SRC_BYREF | RBM_WRITE_BARRIER_DST_BYREF);
+
```

Comment:
This block should be moved before the "Clear the gcInfo" comment and gcMarkRegSetNpt stmt.